src/signature.rs

```diff
@@ -120,43 +120,48 @@ impl Signature {
             return Err(Error::InvalidSignatureLength);
         }
 
-        let r: Uint256 = {
-            let mut data: [u8; 32] = Default::default();
-            data.copy_from_slice(&bytes[0..32]);
-            data.into()
-        };
-        let s: Uint256 = {
-            let mut data: [u8; 32] = Default::default();
-            data.copy_from_slice(&bytes[32..64]);
-            data.into()
-        };
+        let r = Uint256::from_bytes_be(&bytes[0..32]).unwrap();
```

Comment:
same here?

Reply:
there is a guard against `bytes.len() != 65`, so it is safe